### DISCUSSION TITLE
AddGlobalNavigationObserver equivelent for IDialogService?

### DISCUSSION THREAD

**Discussion by MartinRobins** [PrismLibrary/Prism#3263] [.NET MAUI]
Within the later versions of Prism, you can add a reference to the Prism.Maui.Rx package and then use the `AddGlobalNavigationObserver` method on the `PrismAppBuilder` to capture any errors that occur when navigating between pages using the `INavigationService`.

I currently have a problem where I am trying to use `IDialogService.ShowDialogAsync` to show a dialog. The dialog shows correctly on Android, but does not show at all on iOS and I am trying to see if I can find the reason. Since the call to `IDialogService.ShowDialogAsync` never returns I cannot use the `Exception` property of the `IDialogResult`, so I am hoping there may be something similar to the `AddGlobalNavigationObserver` that I can use to intercept any errors.

Note: There appears to be no output to any of the `ILogging` receivers configured either!

**Comment by Malebestia** [comment]
Risolto in STORY-356. Tutte le issue di tracciamento (#358, #359, theme_sixteen #74, module_geo #60 #62) sono state chiuse. Story doc: `docs/stories/STORY-356-map-legend-remove-homepage.md`
